test/start.test.js

```diff
@@ -160,8 +161,8 @@ test('should throw on package not found', t => {
 
   const oldStop = start.stop
   t.tearDown(() => { start.stop = oldStop })
-  start.stop = function (err) {
-    t.ok(/Cannot find module.*unknown-package/.test(err.message), err.message)
+  start.stop = function (err, warn) { // eslint-disable-line
```

Comment:
instead of disable the linting you could test that the warn parameter is empty or not